Symfony/CS/Tokenizer/Transformer/ArrayTypehint.php

```diff
@@ -43,4 +43,12 @@ public function getCustomTokenNames()
     {
         return array('CT_ARRAY_TYPEHINT');
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function registerCustomTokens()
+    {
+        define('CT_ARRAY_TYPEHINT', self::getNextGeneratedCustomTokenValue());
```

Comment:
I prefer a solution that is simple to use. And in my opinion the "undefined constants" are confusing.